internal/services/subscription/validate/location_id.go

```diff
@@ -0,0 +1,21 @@
+package validate
```

Comment:
(as above) we can use the Subscription ID Resource instead so this can be removed

Reply:
Yes. Removed this file.